# Genomics Workshop 2020 August XX - XX

## Location
[Catalina Island Wrigley Institute](https://dornsife.usc.edu/wrigley/)

## Goals
The goals for this workshop were determined by the Steering Committee and include the following:

* Develop new connections among marine biologists, oceanographers, and non-marine evolutionary biologists
* Brainstorm and exchange ideas on ways to bring marine genomics studies up to the forefront of evolutionary thinking
* Develop concensus on best practices in genomics data analysis, defining management units in marine systems, and when understanding adaptation requires knowing the genes
* Produce one or more product(s) based on the discussions from this workshop (product type(s) to be determined at the workshop, but applicants should be prepared to contribute to ongoing discussions after the workshop is over)


## Application
[RCN-ECS Genomics Workshop Application](https://docs.google.com/forms/d/16vNuiXOmvcmsePJvz-6jZAqT3vB4qrpqN4pN-4zCxGw/edit)

The application is due Dec 31, 2019. We have funding to bring together ~50 people. All workshop costs and domestic travel costs will be covered by the RCN. Unfortunately, we do not have funds to reimburse international travel.

There will be three sessions for presentations and in the application there is an opportunity to submit abstracts for one of these sessions. If your application is accepted for the workshop but we are unable to schedule your presentation, we will invite you to present a poster instead.

In addition to presentations, we invite posters on any of the themes highlighted in the schedule.

## Schedule

### Morning 1: Presentations: Hypothesis Testing and Integrative Analyses in Genomics

We invite presentations for testing hypotheses in genomics from a variety of marine and non-marine systems
that go beyond simple "identify the genes." Abstracts that clearly state the hypothesis being tested and the outcome of the test will be given the highest score.

### Afternoon 1: Discussion of filtering, standards, and best practices

High heterozygosity and many types of complex mutations (structural variants, index, and triallelic sites)
appear to be common in marine organisms. These qualities can make *de novo* reference genome assembly 
difficult. In addition, many genomic analyses can be biased by the mapping algorithm, how distant the sample is from the reference, type of sequencing, sources of non-independence in genomic datasets (e.g. related or inbred individuals, complex demography), and choices made in filtering (e.g. thresholds for missing data, which individuals were included, etc.). The best practices for filtering data for analyses are not always clear, as they can be specific to each genome and dataset.

In this discussion we will invite stories and mini-posters about best practicies, what you've learned, pitfalls to watch out for, and "what I wish I knew when I started." Note that we don't want to focus too much on methods that may become outdated in a few years, but we recognize that we need to identify and define some best practices and guidelines for genomics data analysis. Applications that submit stories or ideas for this discussion (in addition to abstracts) will be given a higher score.

### Evening 1: Poster session

### Morning 2: Presentations: Seascape genomes and seascape data

A major hurdle in seascape genomics is the lack of organized environmental/oceanographic data needed to analyze what factors drive reproductive isolation and local adaptation in marine systems.

We invite presentations for seascape genomics studies, oceanography data resources, and marine climate change models (particularly for estuarine and coastal areas). We encourage oceanographers that have access to the data and are interested in developing collaborations with genomicists to apply for the workshop. Applicants that demonstrate knowledge in how to find and use publically available environmental/oceanographic data will be given a higher score.


### Afternoon 2: Discussion of Using genomic data to define management units in marine systems and when the study question requires knowing the genes

How to use genomic data to define management units is not well resolved in marine systems, where lack of differentiation genome-wide does not always imply current connectivity. In addition, local adaptated ecotypes can occur below the scale of gene flow or management unit in marine fishery species. In small breakout groups, we will discuss how to interpret a signals of no genome-wide differentiation interspersed with genomic regions of highly elevated differentiation in the context of defining management units for fisheries and conservation. 

Along similar lines, next generation sequencing is a new technology that can provide a large amount of information, but a large amount of sequencing data is not always informative toward understanding adaptation. In small breakout groups, we will discuss when knowing the genes gives important insights into adaptation, conservation, and other questions that cannot be gleaned from more traditional (e.g., phenotype-based) approaches.

### Evening 2: poster session

### Morning 3: Presentations: Connecting genotype to phenotype with transgenics 

To date, only a few marine species have well developed tools for genetic manipulation. In this session, we invite presentations on the successes and roadblocks in starting a transgenic project in aquatic (marine and non-marine) systems.
Abstracts that have made sufficient progress on a transgenic progress to share results will be given the highest score.

### Afternoon 3: Time for breakout groups, collaborations, etc.

We anticipate that many interesting discussions will arise during the sessions. This afternoon will have open time to continue those discussions. 
